build.sbt

```diff
@@ -32,6 +32,7 @@ lazy val core = crossProject
     shared,
     name := "coursier",
     libs += CrossDeps.scalazCore.value,
+    libs += Deps.jackson,
```

Comment:
I'll try to remove that dependency via extra commits. I don't want to add extra dependencies in the core module, and jackson itself even less.

Reply:
I moved the `Print.scala` and `JsonReport.scala` to a separate module `print-util`, which depends on core, so core does not need to have jackson dep.

Still in progress fixing the CI issues.